addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4911,6 +4911,12 @@ registry.layout_column = SnippetOptionWidget.extend({
             .toggleClass('d-none', !this.$target.is('.s_allow_columns'));
         return this._super(...arguments);
     },
+    /**
+     * @override
+     */
+    cleanForSave() {
+        this._removePaddingPreview();
+    },
```

Comment:
You should probably call super here

Reply:
I looked if I had to but the original `cleanforSave` is empty and the two other present in snippets.options.js don't call super. Should I still do it ?

Reply:
I guess not for cleanForSave 😅  I tend to do it as we never know if the parent class suddenly wants to run a cleanForSave for every option but I assume that would be done in the SnippetEditor instead then.

You can keep it as is 👍🏻